cockroachcloud/cmek.md

```diff
@@ -103,8 +112,7 @@ The CMEK feature has the following limitations:
 
 - CMEK can be enabled only on clusters created after April 1, 2022 (AWS) or June 9, 2022 (GCP).
 - To enable or revoke CMEK on a cluster, you must use the [Cloud API](/docs/cockroachcloud/cloud-api.html). It's not possible to enable CMEK using the {{ site.data.products.db }} Console.
-- If you add a new region to a cluster with CMEK enabled, the new region will not be protected by the CMEK key.
-- Rotating a CMEK key in {{ site.data.products.db }} is not supported. However, if your KMS supports key rotation without changing the key's URI, it will work as expected for a CMEK key.
+- If you add a new region to a cluster with CMEK enabled, the new region will not be automatically protected by the CMEK key.
```

Comment:
What steps would one take to then apply CMEK to that region? Can we link out to those steps?